(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> 'Rows' object has no attribute 'frequencia'"
p3
sS'layer'
p4
S'/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7f04d0386c30, file "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 10>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/leonardo/Downloads/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 197, in <module>\n  File "/home/leonardo/Downloads/web2py/gluon/globals.py", line 419, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 178, in busca_aluno\n    print m.frequencia.nome\nAttributeError: \'Rows\' object has no attribute \'frequencia\'\n'
p11
s.